### PR TITLE
fix json integer parsing

### PR DESCRIPTION
**Pull request by schnipseljagd**
for period_seconds, range_seconds, delay_seconds.

Should fix issue https://github.com/prometheus/cloudwatch_exporter/issues/4.

cc: @fstehle @bracki

### CODE REVIEW

File: src/main/java/io/prometheus/cloudwatch/CloudWatchCollector.java [hunk 1]
```diff
@@ -107,23 +107,23 @@ private CloudWatchCollector(JSONObject config, AmazonCloudWatchClient client) th
             rule.awsStatistics = new ArrayList(Arrays.asList("Sum", "SampleCount", "Minimum", "Maximum", "Average"));
           }
           if (jsonMetricRule.containsKey("period_seconds")) {
-            rule.periodSeconds = (Integer)jsonMetricRule.get("period_seconds");
```
Comment (brian-brazil) [inline src/main/java/io/prometheus/cloudwatch/CloudWatchCollector.java]: There's some up around line 67 that need fixing too

Comment (schnipseljagd) [inline src/main/java/io/prometheus/cloudwatch/CloudWatchCollector.java]: Should be fixed now.

**Comment by brian-brazil** [comment]
Great, thanks!

**Comment by schnipseljagd** [comment]
Thx for merging that fast :100:

**Pull request merged**